Sprint-1/implement/sum.js

```diff
@@ -1,4 +1,13 @@
 function sum(elements) {
+
+ let total = 0;
+
+ for (let i = 0; i <rray.length; i++) {
+    if(typeof arr[i] ==='number') {
+        total += arr[i];
+    }
+ }
+ return total;
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function return the value you expected?

```
sum([NaN, 1]);
sum([Infinity, -Infinity]);
```